api/src/main/kotlin/io/spine/protodata/context/Member.kt

```diff
@@ -159,11 +159,24 @@ protected constructor(
     }
 }
 
+/**
+ * Obtains the proto source file with the given [path].
+ *
+ * @param path The path to the source file which could be relative or absolute.
+ */
+public fun Member<*>.findSource(path: File): ProtobufSourceFile? {
+    return (this as Querying).select<ProtobufSourceFile>().findById(path)
```

Comment:
It is a single line expression, you can save on braces.

And please, mention when it is OK for this function to return `null`. The same suggestion is to the function below.